lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
Similar issue here as in https://github.com/openmicroscopy/bioformats/pull/2264/files#r54471656 that this repo may not should know about `OME_FILES`.